client/src/app.css

```diff
@@ -25,6 +25,15 @@
   pointer-events: none;
 }
 
+#piece-hover-tooltip {
```

Comment:
Please use [https://tailwindcss.com/](tailwind) and LMK if you have any problems.  In some cases, you may want to go custom CSS for something, but there is a way to define a custom class if you need that.  Let's avoid using `app.css`.

Reply:
FYI, looks like this loosely translates to

`p-2 hidden fixed overflow-hidden border border-solid border-black rounded`

Reply:
Sounds good, had never heard of tailwind. See [94cf71c](https://github.com/trumpet-zk-ignite-1/mina-arena/pull/19/commits/94cf71ca5c1ae4cd99cd3ed4321ea604ffd39a58) where I switch to use that instead.